docs/sources/shared/upgrade/upgrade-common-tasks.md

```diff
@@ -6,62 +6,6 @@ labels:
 title: Upgrade guide common tasks
 ---
 
-We recommend that you upgrade Grafana often to stay current with the latest fixes and enhancements.
```

Comment:
I'd put a note here recommending that users back up before any upgrade and linking to the new doc

Reply:
Ah, the upgrade guide contains all backup instructions, so those instructions are published to two places. We created a shared snippet and added it to the upgrade guide and to the backup topic.

Reply:
Chatted with Dan, we determined this isn't necessary.